src/applet.c

```diff
@@ -323,11 +319,20 @@ static void hotkey_cb(__brisk_unused__ GdkEvent *event, gpointer v)
 static void brisk_menu_applet_settings_changed(GSettings *settings, const gchar *key, gpointer v)
 {
         BriskMenuApplet *self = v;
+        gchar *value = NULL;
+        gchar *transl = NULL;
+
+        value = g_settings_get_string(settings, key);
 
         if (g_str_equal(key, "hot-key")) {
-                brisk_menu_applet_update_hotkey(self, g_settings_get_string(settings, key));
+                brisk_menu_applet_update_hotkey(self, value);
         } else if (g_str_equal(key, "label-text")) {
-                brisk_menu_applet_update_label_text(self, g_settings_get_string(settings, key));
+                if (strlen(value) > 0) {
+                        gtk_label_set_text(GTK_LABEL(self->label), value);
+                } else {
+                        transl = _("Menu");
```

Comment:
This seems unnecessary :) Can still us `_("Menu")` directly  without holding a temporary ref